System/Directory.hs

```diff
@@ -402,6 +402,9 @@ removeDirectory = removePathInternal True
 -- symbolic links are removed without affecting their targets.
 --
 -- On Windows, the operation fails if /dir/ is a directory symbolic link.
+--
+-- This operation is reported to be flaky on Windows so retry logic may be advisable.
+-- See: [github.com/haskell/directory/pull/108](https://github.com/haskell/directory/pull/108)
```

Comment:
I believe the Haddock syntax for links is a bit different: https://www.haskell.org/haddock/doc/html/ch03s08.html#idm140354810727840

That being said probably simpler to just write out the link directly:

    https://github.com/haskell/directory/pull/108

Not much is being saved by shaving off the `https://`.

Reply:
Great, the link you sent also says:

> If Haddock sees something that looks like a URL (such as something starting with http:// or ssh://) where the URL markup is valid, it will automatically make it a hyperlink.

I've removed the mark-up.